web-marketplace/src/pages/Marketplace/Storefront/Storefront.normalizer.ts

```diff
@@ -32,28 +30,6 @@ export const normalizeProjectsInfosByHandleMap = ({
     { name: string; classIdOrName: string; classId: string }
   >();
 
-  offChainProjects?.nodes?.forEach(project => {
```

Comment:
not really related to /projects or individual project pages, but since I've modified web-marketplace/src/graphql/AllProjects.graphql query which was being used here, I've realized that this seems quite outdated and not needed